os/kernel/debug/mem_leak_checker.c

```diff
@@ -89,9 +99,11 @@ static void add_hash(int index)
 static bool search_hash(unsigned long value)
 {
 	long key = value % HASH_SIZE;
-	struct alloc_node_info_s *cur = hash_table[key];
+	struct alloc_node_info_s *cur;
+
+	cur = hash_table[key];
```

Comment:
you can asign directly as previous code. and now it use direct allocated memory, so null check for cur also need to be added in line 106

Reply:
Thanks, changed